en/hosting_guide/server_provisionning.rst

```diff
@@ -0,0 +1,77 @@
+Server provisionning
+====================
+
+Context
+-------
+
+Believe it or not, at some time, your Open Orchestra project will require an external
+server to be deployed on.
```

Comment:
Matter of taste, but i'm not a fan of the way the sentence is written. (Believe it or not, etc.)